tests/core.py

```diff
@@ -456,6 +457,40 @@ def trigga(context, obj):
             dag=self.dag)
         t.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, ignore_ti_state=True)
 
+        session = settings.Session()
+        new_dag_run = session.query(models.DagRun).filter(
+            models.DagRun.run_id == trigga.run_id).first()
+        self.assertEqual(new_dag_run.execution_date, DEFAULT_DATE)
+
+    def test_trigger_dagrun_order_modified(self):
+        """
+        Test TriggerDagRunOperator with changes in DagRunOrder
+        """
+        new_execution_date = datetime(2016, 1, 1)
+        new_dag_run_id = 'manual_run_id'
+        payload_key = 'message'
+        payload = {payload_key: 'Hello World'}
+
+        def trigga(context, obj):
+            obj.run_id = new_dag_run_id
+            obj.execution_date = new_execution_date
+            obj.payload = payload
+            if True:
```

Comment:
Yes. I have tried to respect as much I could the previus TriggerDagRun test. If you see the original `test_trigger_dagrun` method, you will see the same sentence `if True`.
I think the original author's intentions were to make it clear that the `python_callable` function can check and decide if the dagrun shoud be triggered. But it's just a theory.